File: 01703.cpp
// 생장점

#include <iostream>
using namespace std;

int main() {
	while (1) {
		int a; cin >> a;
		if (a == 0) break;

		int leaf = 1;
		for (int i = 0; i < a; i++) {
			int sf, br; cin >> sf >> br;
			leaf *= sf;
			leaf -= br;
		}

		cout << leaf << endl;
	}
}

File: 01703.py
while True:
    branches = list(map(int, input().split()))
    a = branches[0]
    if a == 0:
        break

    leaf = 1
    for i in range(1, a * 2 + 1, 2):
        leaf *= branches[i]
        leaf -= branches[i + 1]

    print(leaf)
